src/orion/core/worker/experiment.py

```diff
@@ -580,7 +592,9 @@ def configure(self, config, enable_branching=True, enable_update=True):
             # will be used when EVC is not available.
             # must_branch = self._is_different_from(experiment.configuration)
             branching_configuration = fetch_branching_configuration(config)
-            conflicts = detect_conflicts(self.configuration, experiment.configuration)
+            configuration = self.configuration
+            configuration['_id'] = self._id
```

Comment:
Where do we need this _id?

Reply:
ok, found: https://github.com/Epistimio/orion/blob/develop/src/orion/core/evc/conflicts.py#L1611